Undo the broken formatting

heesunlee9 opened the pull request:
I mistakenly broke formatting in '4. On the Create namespace page, take the following steps:'.

Viewing the original document, (https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create)
I undo the broken formatting as the original document.

heesunlee9 committed:
Undo the broken formatting 

I mistakenly broke formatting in '4. On the Create namespace page, take the following steps:'
viewing the original document(https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create)
i undo the broken formatting as the original document.

spelluru reviewed (commented):
@heesunlee9 - When I view file, the formatting is still wrong. I don't see a couple of images. Could you send me the summary of changes you want to make? I will make those changes to the article. Thanks.

heesunlee9 committed:
Make two pictures visible

Make two pictures visible

       ![Create an event hub namespace](./media/event-hubs-quickstart-portal/create-event-hub1.png)
       ![Review + create page](./media/event-hubs-quickstart-portal/review-create.png)

spelluru reviewed (approved):
Thanks!

ktoliver committed:
public repo edit
